donor-registry/src/app/certificate/certificate.component.ts

```diff
@@ -56,8 +57,18 @@ export class CertificateComponent implements OnInit {
     private config: AppConfig) {
   }
   ngOnInit(): void {
+    console.log(screen.orientation.angle);
+   if(screen.orientation.angle == 90){
+    this.orientation = "_landscape"
```

Comment:
@chaitrali-r add condtion like below

this.orientation= (orientation == 90) ?  "_landscape" : '_portrait'